solutions/add_two_numbers.ts

```diff
@@ -0,0 +1,17 @@
+import { SinglyLinkedListNode } from '../types/linked_list.ts';
+
+// 2. Add Two Numbers
+// https://leetcode.com/problems/add-two-numbers/
+export default function addTwoNumbers(nodeA: SinglyLinkedListNode<number>, nodeB: SinglyLinkedListNode<number>) {
+  if (!nodeA || !nodeB) return nodeA ?? nodeB ?? null;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/axross/leetcode-typescript/pullRequest?prid=7152801)
```suggestion
  if (!nodeA || !nodeB) {return nodeA ?? nodeB ?? null;}
```